.github/workflows/report-exchange.yml

```diff
@@ -0,0 +1,61 @@
+name: Exchange
+
+on: workflow_dispatch
+
+jobs:
+  credentials_exchange:
+    runs-on: ubuntu-latest
+    strategy:
+      # Hard-coding a single test entry, this will change to include all
+      # permutations of alice/bob actors.
```

Comment:
Note that this matrix is written to function with local `act` testing, which doesn't seem to like the `include:` variant. It will likely morph into something more like:
```
matrix:
  include:
    - alice: MESUR_PRODUCTION
      bob: TRANSMUTE_STAGING
    - alice: TRANSMUTE_STAGING
      bob: MESUR_PRODUCTION
```